fern/changelog/05-01-2025.mdx

```diff
@@ -0,0 +1,9 @@
+## Summary
+
+Header API Key In List Grant APIs
+
+### What's new?
+
+We've remove the `x-chariot-api-key` header from the List Grants API as a requirement. In the past, this header was used to filter the list of grants by a specific Connect. This is no longer necessary as the `connect_id` query parameter can be used for this purpose instead.
```

Comment:
can we add a sentence about being able to omit this parameter and get back the full list of grants that you are entitled to see across all Connect objects